composer.json

```diff
@@ -52,7 +52,7 @@
         "oat-sa/extension-tao-revision": "10.2.0",
         "oat-sa/extension-tao-mediamanager": "12.21.2",
         "oat-sa/extension-pcisample": "3.4.0",
-        "oat-sa/extension-tao-backoffice": "6.6.1",
+        "oat-sa/extension-tao-backoffice": "6.6.2",
```

Comment:
We are missing composer.lock

Reply:
:facepalm: (on it)

Reply:
:point_right: 3e98ef02023efa204ffa562159379fa82d4e712e